docs/luigi-client-api.md

```diff
@@ -189,8 +189,16 @@ Removes the loading indicator. Use it after calling [showLoadingIndicator()](#sh
 
 ### addBackdrop
 
-Adds a backdrop to block the top and side navigation. It is based on Fundamental UI Modal, which you can use in your micro front-end to achieve the same behavior.
+Adds a backdrop to block the top and side navigation. It is based on the Fundamental UI Modal, which you can use in your micro front-end to achieve the same behavior.
 
 ### removeBackdrop
 
 Removes the backdrop.
+
+### setDirtyStatus
+
+Makes current page dirty or not
```

Comment:
```suggestion
Makes current page dirty or not.
```

Reply:
Could you explain in more details what is meant in this sentence? As for the periods at the end of the sentence and starting it with the verb without a subject I am against but added it here to keep the consistency in the whole text as I can see the rest starts and end similarly.

Reply:
This method informs the main application that the current page in the iframe is dirty, like for example a page with form fields which has been modified from their original values.

Reply:
done